gwpy/timeseries/tests/test_core.py

```diff
@@ -107,26 +107,45 @@ def test_epoch(self):
             a.t0, units.Quantity(1126259462, 's'))
 
     def test_sample_rate(self):
-        """Test `gwpy.timeseries.TimeSeriesBase.sample_rate`
+        """Test `gwpy.timeseries.TimeSeriesBase.sample_rate`.
         """
         # check basic conversion from dt -> sample_rate
         a = self.create(dt=0.5)
         assert a.sample_rate == 2 * units.Hz
 
+    def test_sample_rate_del(self, array):
+        """Test that `sample_rate` cannot be deleted.
+        """
         # test that we can't delete sample_rate
         with pytest.raises(AttributeError):
-            del a.sample_rate
+            del array.ample_rate
```

Comment:
ample_rate?

Reply:
That will definitely raise an `AttributeError`, good catch.